vignettes/wwinference.Rmd

```diff
@@ -283,9 +296,8 @@ set the MCMC settings using `get_mcmc_options()`, and pass in our
 pre-compiled model(`model`) to `wwinference()` where they are combined and
 used to fit the model.
 
-
-```{r warning=FALSE, message=FALSE}
-fit <- wwinference::wwinference(
+```{r fitting-model, warning=FALSE, message=FALSE}
+ww_fit <- wwinference::wwinference(
```

Comment:
Can we call the `print()` method and `summary()` method to demonstrate how they work?